src/elm/Page/Community/Settings/Features.elm

```diff
@@ -298,6 +298,7 @@ saveFeature feature state authorization { shared, accountName } community =
             , description = community.description
             , inviterReward = community.inviterReward
             , invitedReward = community.invitedReward
+            , minBalance = community.minBalance |> Maybe.withDefault -100
```

Comment:
Same here: `-100` is used as the default value but in the form for the new community `-10` is used.

Reply:
I'll change the default on the form and keep it like this